models/root/couchdb.sql

```diff
@@ -0,0 +1,13 @@
+{{
+    config(
+        materialized = 'view',
+        indexes=[
+            {'columns': ['"@timestamp"'], 'type': 'brin'},
```

Comment:
Unsure if this index is actually applied since we are creating a view. We should probably move this to the underlying table